test/lens_test.exs

```diff
@@ -37,6 +37,27 @@ defmodule LensTest do
     end
   end
 
+  describe "otherwise" do
+    test "to_list" do
```

Comment:
Maybe something like `get_in(%{a: 1}, [either(Lens.key?(:a), Lens.key?(:b))])` vs `get_in(%{b: 2}, [either(Lens.key?(:a), Lens.key?(:b))])` would be the easiest example? Also, move this test into doctests, where it can serve as documentation as well :)